conf/jenkins/configuration.yaml

```diff
@@ -0,0 +1,55 @@
+jenkins:
+  agentProtocols:
+    - "JNLP4-connect"
+    - "Ping"
+  numExecutors: 0
+  clouds:
+    - mesos:
+        agentUser: "${JENKINS_AGENT_USER:-root}"
+        frameworkName: "${JENKINS_FRAMEWORK_NAME:-Jenkins Scheduler}"
+        jenkinsURL: "http://${JENKINS_FRAMEWORK_NAME:-jenkins}.${MARATHON_NAME:-marathon}.mesos:${PORT0:-8080}"
+        mesosAgentSpecTemplates:
+          - label: "${JENKINS_LINUX_AGENT_LABEL:-linux}"
+            agentAttributes: "${JENKINS_LINUX_AGENT_OFFER_SELECTION_ATTRIBUTES:-}"
+            agentCommandStyle: Linux
+            containerInfo:
+              dockerForcePullImage: false
+              dockerImage: "${JENKINS_LINUX_AGENT_IMAGE:-mesosphere/jenkins-dind:0.6.0-alpine-experimental}"
```

Comment:
Out of curiosity, why  are we using a dind image as default and not a simple official JDK image?

Reply:
The common use case for customers is to build docker images on the jenkins agents, the simple JDK image doesn't do this and will break the default workflow for many users.